coalib/results/Diff.py

```diff
@@ -200,9 +225,12 @@ def unified_diff(self):
         for equality comparison.
         """
         return ''.join(difflib.unified_diff(
-            self.original,
-            self.modified,
-            tofile=self.rename if isinstance(self.rename, str) else ''))
+                       self.original,
+                       self.modified,
+                       tofile=self.rename if isinstance(self.rename, str)
+                       else self.create
+                       if isinstance(self.create, str)
+                       else ''))
```

Comment:
as per https://github.com/coala/coala/pull/3610#pullrequestreview-27982920 I changed to this style again, I think the one you commented was there before you asked for reverting back to this one. :/